projects/GKv3/GKComponents/GKUI/Components/HyperView.cs

```diff
@@ -333,30 +335,55 @@ private void DoPaint(Graphics gfx)
                 } finally {
                     fAcceptFontChange = true;
                     if (brush != null) brush.Dispose();
-                    if (font != null) font.Dispose();
                 }
             } catch (Exception ex) {
                 Logger.WriteError("HyperView.DoPaint()", ex);
             }
         }
 
-        private Font ProcessFont(Font prevFont, float emSize, FontStyle style)
+        private Font ProcessFont(Font prevFont, float emSize, FontStyle style, FontDecoration decoration)
         {
             Font result;
             if (prevFont == null) {
                 var defFont = this.Font;
-                result = new Font(defFont.FamilyName, emSize, style);
+                result = new Font(defFont.FamilyName, emSize, style, decoration);
             } else {
-                if (prevFont.Size == emSize && prevFont.FontStyle == style) {
+                if (prevFont.Size == emSize && prevFont.FontStyle == style && prevFont.FontDecoration == decoration) {
                     result = prevFont;
                 } else {
-                    result = new Font(prevFont.FamilyName, emSize, style);
-                    prevFont.Dispose();

```

Comment:
Fonts have shared handler, so disposing one font breaks fonts with the same family. Do not dispose fonts.

Reply:
👍